src/components/keyboard-custom/keyboard-custom.tsx

```diff
@@ -19,7 +19,7 @@ export interface IProps {
     showNumeric?: boolean;
     handleTextUpdate: (key: any) => void;
     handleDeleteKey: () => void;
-    buttons?: Array<{ label: string; onPress: () => void; style?: {}; disabled?: boolean }>;
+    buttons?: { label: string; onPress: () => void; style?: {}; disabled?: boolean }[];
```

Comment:
something is weird here :))) @maricalucian is adding Array you are deleting it :))

Reply:
:))) I think it's because we have different typescript versions (maybe the global ones). I have seen then when I commit & push from Github Desktop, the code automatically changes from `Array` to `[ ]` :)) 

I have
```
tsc --version
Version 3.7.5
```